docs/specifications/cva6_requirement_specification.adoc

```diff
@@ -0,0 +1,391 @@
+
+It is an industrial evolution of ARIANE created by ETH Zürich and the University of Bologna. It is written in SystemVerilog and maintained by the OpenHW Group.
+
+This specification is organized as requirements that apply to the “Scope of the IP”.
+
+The requirement list is to be approved by the OpenHW Group Technical Work Group (TWG), as well as its change requests.
+
+The specification will be complemented by a user’s guide.
+
+Revision 0 refers to the product of the first CVA6 project led at OpenHW Group. It is a placeholder in case of future evolutions after project freeze (PF gate).
+
+A list of abbreviations is available at the end of this document.
+
+== Scope
+
+=== Scope of the IP
+
+The *scope of the IP* is the subsystem that is specified below and that will undergo verification with a 100% coverage goal. In the verification plans, the scope of the IP can be broken down in several DUT (design under test).
+
+The scope of the IP is the *CVA6 hardware* supporting all the features used in products based on CVA6.
+
+CVA6 exists in two main configurations: *CV64A6* and *CV32A6*. A requirement referring to CVA6 applies to both configurations.
+
+image::images/cva6_scope.png[]
+
+As displayed in the picture above, the IP comprises:
+
+* The CVA6 core;
+* L1 write-through cache;
+* Optional FPU;
+* Optional MMU;
+* Optional PMP;
+* CSR;
+* Performance counters;
+* Interface with the P-Mesh coherence system of OpenPiton.
+
+These are not part of the IP (several solutions can be used):
+
+* CLINT or PLIC Interrupt modules;
+* Debug module (such as DTM);
+* Support of L1 write-back cache (this might come later as an update).
+
+In addition to these main configurations, several fine grain parameters are available.
+
+Unless otherwise stated, an optional feature is controlled by a SystemVerilog parameter. If not selected, the optional feature will not be present in the netlist after synthesis.
+
+The reader’s attention is drawn to the difference between an optional feature (“...shall support as an option...”) and a desired goal (“...should support...”, “....should reduce latency...”).
+
+These are not in the scope of this specification:
+
+* SW layers, such as compiler and OSes (that can however be part of the OpenHW Group CVA6 project);
+* SW emulation of RISC-V optional extensions ( feasible but the scope of the IP is the core hardware);
+* Other features included in the testbench (main memory, firmware, interconnect…), the coverage of which will not be measured;
+* The vector coprocessor (CV-VEC) that is planned to interface with CV64A6.
+
+=== Verified configurations
+
+It is not possible to verify all 2^n^ combinations of parameters. Below is the list of configurations that will undergo verification in the project and their main parameters. The full list of parameters for each configuration will be detailed in the users’ guide.
```

Comment:
Probably not really 2^n as some of the parameters aren't binary. Anyway, it is possible.  I'd rather say "it is not feasible to verify all possible combinations of parameters".

Reply:
Duplicate remark.